client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -170,25 +172,39 @@ class EditorDiffViewer extends PureComponent {
 		const countAbove = this.changesAboveViewport.length;
 		const countBelow = this.changesBelowViewport.length;
 
+		const fields = [];
+
+		for ( const field in diff ) {
+			// Skip post_title field.
+			if ( 'post_title' === field ) {
+				continue;
+			}
+
+			fields.push(
+				<div key={ field }>
+					<h2 className="editor-diff-viewer__fieldname">{ revisionFields[ field ] }</h2>
```

Comment:
Would it be possible to hide this title if we don't have revisions for footnotes?

Most of the time, we'll only have revisions for the content with no revisions for footnotes, so I was wondering if for these cases we can keep the current UI that does not display a "Content" title:

Desired | Actual
--- | ---
<img width="783" alt="Screenshot 2024-08-20 at 19 38 51" src="https://github.com/user-attachments/assets/b2ca484c-2c25-4be8-a86a-db38442bcabe"> | <img width="785" alt="Screenshot 2024-08-20 at 19 39 22" src="https://github.com/user-attachments/assets/fad3fc08-7f23-4a0b-9b20-0bcd6918d202">

Reply:
If we want to be more aligned with core, then we should also add a heading for the title 😄 

My main concern is that, without any other headings, some users might think that "Content" is part of the post content rather than a heading, and that can cause some confusion.

I'm not opposed to leave it as-is right now, but can you maybe check with @javierarce or @lucasmendes-design before?

Reply:
I agree with @mmtr. I think it's easy to see in Core that there are two sections (title and content) that are not part of the changes. The 'removed' and 'added' labels also help to reinforce that perception.

So, I think adding a title section as Miguel suggests could help. Alternatively, we could add a slight gray border to the bottom of the 'Content' label to separate it from the changes.